ee/models/license.py

```diff
@@ -65,21 +65,31 @@ def available_features(self) -> List[str]:
         return self.PLANS.get(self.plan, [])
 
 
+def get_max_users() -> Optional[int]:
+    """
+    Returns the maximum number of users allowed.
+    Examines all available valid licenses and returns the max users available.
+    """
+    licenses = License.objects.filter(valid_until__gte=timezone.now())
+    if len(licenses) > 0:
+        return max([l.max_users for l in licenses])
```

Comment:
This is incorrect - if any licenses are unlimited, this will blow up.

```python
user_limits = [l.max_users for l in License.objects.filter(valid_until__gte=timezone.now())]
if None in user_limits:
  return None
return max(user_limits, default=None)
```